### ISSUE TITLE
cuelang.org: code-tabs cannot be used with a step

### ISSUE THREAD

**Issue by myitcv** [cue-lang/cue#3005]
### What page were you looking at?

https://cl-1191217-3--cue-cls.netlify.app/docs/tutorial/working-with-the-central-registry/

### What version of the site were you looking at?

https://github.com/cue-lang/cuelang.org/commit/fa8bde9aa766cd3415982b63a48cce390fafa77c

### What did you do?

https://review.gerrithub.io/c/cue-lang/cuelang.org/+/1191217/3

### What did you expect?

Well rendered pages that have multi-step scripts

### What did you see instead?

https://cl-1191217-3--cue-cls.netlify.app/docs/tutorial/working-with-the-central-registry/

In particular, look at how the upload nodes fail to render properly:

![Screenshot 2024-04-03 at 18 54 40](https://github.com/cue-lang/cue/assets/3374574/3afc0efe-2830-4450-b741-4bd2a9b7dd90)

**Comment by anne-usmedia** [comment]
@myitcv This looks like an issue with the (absence of) empty lines between the shortcodes. 

If I add the shortcodes likes this:
```
{{< step stepNumber="4" >}}

Create the code for the new module:

{{< code-tabs >}}

{{< code-tab name="config.cue" language="cue">}}
package frostyapp

import "github.com/cue-labs/examples/frostyconfig@v0"

config: frostyconfig.#Config & {
	appName: "alpha"
	port:    80
	features: logging: true
}
{{< /code-tab >}}

{{< /code-tabs >}}
```

I get the following results:
![Screenshot 2024-05-30 at 12 30 01](https://github.com/cue-lang/cue/assets/122522301/9407e03f-54c8-4566-8145-9f0a55d3a1e2)

Can you try if this works for you?

**Comment by jpluscplusm** [comment]
> Can you try if this works for you?

@anne-usmedia I've put your example code on a new page, and I still get this breakage:

![image](https://github.com/cue-lang/cue/assets/224508/c21a5b9f-9271-41a1-9f88-35ebe289ef7e)

I've confirmed that, whilst I'm placing this inside the `content` directory, the preprocessor isn't doing anything to this test content, as it emits it into the `hugo` hierarchy:

```
$ md5sum content/examples/step-whitespace/en.md  hugo/content/en/examples/step-whitespace/index.md
bd939f2ed419665b4428a28fd918e7d3  content/examples/step-whitespace/en.md
bd939f2ed419665b4428a28fd918e7d3  hugo/content/en/examples/step-whitespace/index.md
```

I'm using:
- Hugo version `hugo v0.126.1-3d40aba512931031921463dafc172c0d124437b8+extended linux/amd64 BuildDate=2024-05-15T10:42:34Z VendorInfo=gohugoio`
- and cuelang.org commit `d155425d5afade1de12cfb4cbbc64ad4fdb95d87` (current tip of master)

**Comment by jpluscplusm** [comment]
Test source, to help debugging: 

```markdown
---
title: Test of whitespace around step markup
---

{{< step stepNumber="4" >}}

Create the code for the new module:

{{< code-tabs >}}

{{< code-tab name="config.cue" language="cue">}}
package frostyapp

import "github.com/cue-labs/examples/frostyconfig@v0"

config: frostyconfig.#Config & {
    appName: "alpha"
    port:    80
    features: logging: true
}
{{< /code-tab >}}

{{< /code-tabs >}}

{{</step>}}
```

**Comment by myitcv** [comment]
Confirmed that this is fixed as of https://cl-1191217-5--cue-cls.netlify.app/docs/tutorial/working-with-the-central-registry/.

**Comment by myitcv** [comment]
Fixed via https://review.gerrithub.io/c/cue-lang/cuelang.org/+/1195633
